docs/core/1.2/cli/sawadm.md

```diff
@@ -35,34 +51,23 @@ initialization of a validator. A network requires an initial block
 determine the blockchain ID. This initial block is produced from a list
 of batches, which will be applied at genesis time.
 
-\<\<\<\<\<\<\< HEAD:docs/core/1.1/cli/sawadm.rst The optional argument
-[input_file]{.title-ref} specifies one or more files containing
-serialized `BatchList` protobuf messages to add to the genesis data.
-(Use a space to separate multiple files.) If no input file is specified,
-this command produces an empty genesis block. ======= The
-[input_file]{.title-ref} argument specifies one or more files containing
-serialized `BatchList` protobuf messages to add to the genesis data.
-(Use a space to separate multiple files.) At least one input file must
-be specified and it must contain the required settings, unless the
-`--ignore-required-settings` argument is used (run
-`sawadm genesis --help` for more info). \>\>\>\>\>\>\>
-core/1-2:docs/core/1.2/cli/sawadm.rst
+The [input_file]{.title-ref} argument specifies one or more files containing
```

Comment:
.title-ref is a sphinx-ism I think

Reply:
Fixed in all files I had already worked in